CHANGELOG.rst

```diff
@@ -1,6 +1,23 @@
 Changelog
 ==========
 
+.. _2.7.0:
+
+2.7.0 - 2024-05-29
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- New features:
+    - Fix the User endpoint acording to openAPI standar .
```

Comment:
Some typos here: 
- acording => according
- standar => standard
- troubleshouting => troubleshotting

Reply:
Incredibly minor but I think OpenAPI is usually stylised with capital first letter.